pom.xml

```diff
@@ -245,7 +245,7 @@
         <dependency>
             <groupId>org.fenixedu</groupId>
             <artifactId>fenixedu-spaces</artifactId>
-            <version>1.3.1</version>
+            <version>1.4.0-SNAPSHOT</version>
```

Comment:
If you REALLY depend on this new version, wait for it to be released before changing it in the POM.